contracts/NFTVesting/README.md

```diff
@@ -0,0 +1,21 @@
+# NFT Smart Contract Vesting
+
+- These set of contracts is based on the [article]() 
+
+
+
+### Setup
+
+### Dependencies
+- Hardhat
+
+
+### Commands to Deploy
+For testing of the contracts, you can refer to `scripts/NFTVesting/deploy.js`.
+
+1. Start local chain with `npx hardhat node`
+2. Run the deploy script with
```

Comment:
Enumerate for all the different cases for now

Don't worry about the automated tests, will get one of our Spartan Labs tech team to turn these scripts into automated tests as a form of capability building for the team

Reply:
got it!

Reply:
@ExtremelySunnyYK update the below to enumerate the different files for each section, happy and unhappy etc. for each lock contract